taui/configurations/default/messages.yml

```diff
@@ -117,6 +120,11 @@ Profile:
   NameRequired: Please enter a name for the head of household.
   SaveError: Failed to save profile. Please try again.
   Title: Profile
+ImportanceLabels:
+  1: Not important
```

Comment:
I re-read the issue discussion and I'm guessing this is to prevent the user from entering all zero weights. I probably would have opted to special-case the all-zeros case, but preventing people from completely ignoring any factor might lead to more intuitive behavior because increasing or decreasing a single weight will always result in some kind of change. So I think this is fine for now. If / when we revisit this project we can tweak the behavior if it turns out to be confusing.

Reply:
I didn't include zero here to prevent ambiguity between zero and unset values, if the profile predates adding these fields. We can adjust to a zero-based range during calculation, if needed. The numeric values aren't user facing, and so can be arbitrary.